azurerm/internal/services/healthcare/healthcare_service_data_source_test.go

```diff
@@ -24,12 +24,26 @@ func TestAccHealthCareServiceDataSource_basic(t *testing.T) {
 				check.That(data.ResourceName).Key("resource_group_name").Exists(),
 				check.That(data.ResourceName).Key("kind").Exists(),
 				check.That(data.ResourceName).Key("cosmosdb_throughput").Exists(),
+				check.That(data.ResourceName).Key("cosmosdb_key_vault_key_versionless_id").DoesNotExist(),
```

Comment:
With the above, I think we need to check for a zero value here

```suggestion
				check.That(data.ResourceName).Key("cosmosdb_key_vault_key_versionless_id").HasValue(""),
```